src/attribution/onINP.ts

```diff
@@ -37,8 +37,12 @@ const attributeINP = (metric: INPMetric): void => {
       );
     })[0];
 
+    // TODO: remove when 1367329 is resolved
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=1367329
+    const targetEntry = metric.entries.find(entry => entry.target);
```

Comment:
Actually let's be a bit more explicit as to why we're doing this, as this is a reference library.

```suggestion
    // Currently Chrome can return a null target for certain event types
    // (especially pointer events). As the event target should be the same
    // for all events in the same interaction, we pick the first non-null one.
    // TODO: remove when 1367329 is resolved
    // https://bugs.chromium.org/p/chromium/issues/detail?id=1367329
    const firstTargetEntry = metric.entries.find(entry => entry.target);
```